tests/test_models.py

```diff
@@ -289,3 +295,52 @@ class TestRefreshTokenModel(TestCase):
     def test_str(self):
         refresh_token = RefreshToken(token="test_token")
         self.assertEqual("%s" % refresh_token, refresh_token.token)
+
+
+class TestClearExpired(TestCase):
+
+    def setUp(self):
+        self.user = UserModel.objects.create_user("test_user", "test@example.com", "123456")
+        """Insert two tokens on database."""
+        AccessToken.objects.create(
+            id=1,
+            token='555',
+            expires=dt.now(),
+            scope=2,
+            application_id=3,
+            user_id=1,
+            created=dt.now(),
+            updated=dt.now(),
+            source_refresh_token_id='0')
+        AccessToken.objects.create(
+            id=2,
+            token='666',
+            expires=dt.now(),
+            scope=2,
+            application_id=3,
+            user_id=1,
+            created=dt.now(),
+            updated=dt.now(),
+            source_refresh_token_id='1')
+
+    def test_clear_expired_tokens(self):
+        oauth2_settings.REFRESH_TOKEN_EXPIRE_SECONDS = 60
+        assert clear_expired() is None
+
+    def test_clear_expired_tokens_incorect_timetype(self):
+        oauth2_settings.REFRESH_TOKEN_EXPIRE_SECONDS = 'A'
+        with pytest.raises(ImproperlyConfigured) as excinfo:
+            clear_expired()
+        result = excinfo.value.__class__.__name__
+        assert result == 'ImproperlyConfigured'
+
+    def test_clear_expired_tokens_with_tokens(self):
```

Comment:
[Question] Does this test depend on it taking >= 1 second between object creation and the test case? If so, then it will fail if the test itself takes < 1 second?

Reply:
maybe, then i change this value to 0